pages/_app.tsx

```diff
@@ -12,12 +12,17 @@ const App: React.FC<AppProps> = ({ Component, pageProps }) => {
   return (
     <ApolloProvider client={apolloClient}>
       <Provider session={pageProps.session}>
-        <CSSReset />
-        <Head>
-          <meta content="width=device-width, initial-scale=1" name="viewport" />
-        </Head>
-        <GlobalStyle />
-        <Component {...pageProps} />
+        <ChakraProvider>
```

Comment:
The way you had to do it for ssr is have the provider around the pages per their docs which ultimately causes the flash, so just went back to this way which introduces a issue for server side pages, but we'll cross that bridge when the time comes!